src/commonMain/kotlin/fr/acinq/eclair/transactions/Transactions.kt

```diff
@@ -94,7 +117,12 @@ object Transactions {
         data class HtlcPenaltyTx(override val input: InputInfo, @Serializable(with = TransactionKSerializer::class) override val tx: Transaction) : TransactionWithInputInfo()
 
         @Serializable
-        data class ClosingTx(override val input: InputInfo, @Serializable(with = TransactionKSerializer::class) override val tx: Transaction) : TransactionWithInputInfo()
+        data class ClaimHtlcDelayedOutputPenaltyTx(override val input: InputInfo, @Serializable(with = TransactionKSerializer::class) override val tx: Transaction) : TransactionWithInputInfo()
+
+        @Serializable
+        data class ClosingTx(override val input: InputInfo, @Serializable(with = TransactionKSerializer::class) override val tx: Transaction, val toLocalIndex: Int?) : TransactionWithInputInfo() {
+            val toLocalOutput: TxOut? get() = toLocalIndex?.let { tx.txOut.getOrNull(it) }
```

Comment:
```suggestion
            val toLocalOutput: TxOut? get() = toLocalIndex?.let { tx.txOut[it] }
```
if `toLocalIndex` is set it must point to an existing output

Reply:
Done in https://github.com/ACINQ/eclair-kmp/pull/229/commits/4b46d6121879bb9ef82542100ae63e5a1fd62ec4